File: Python/7kyu - Building blocks.py
class Block:
    def __init__(self, data):
        width, length, height = data
        self.width = width
        self.length = length
        self.height = height
        self.volume = width * length * height
        self.surface_area = 2*(width*length + width*height + length*height)

    def get_width(self):
        return self.width

    def get_length(self):
        return self.length

    def get_height(self):
        return self.height

    def get_volume(self):
        return self.volume

    def get_surface_area(self):
        return self.surface_area

File: C#/7kyu - Building blocks.cs
class Block
{
  private int Width { get; }
  private int Length { get; }
  private int Height { get; }
  private int Volume { get; }
  private int SurfaceArea { get; }
  
  public Block(int[] data)
  {
    Width = data[0];
    Length = data[1];
    Height = data[2];
    Volume = Width * Length * Height;
    SurfaceArea = 2 * (Width * Length + Width * Height + Length * Height);
  }
  
  public int GetWidth() => Width;
  public int GetLength() => Length;
  public int GetHeight() => Height;
  public int GetVolume() => Volume;
  public int GetSurfaceArea() => SurfaceArea;
}
